src/library.cpp

```diff
@@ -149,8 +149,10 @@ void *lammps_extract_global(void *ptr, char *name)
   if (strcmp(name,"xz") == 0) return (void *) &lmp->domain->xz;
   if (strcmp(name,"yz") == 0) return (void *) &lmp->domain->yz;
   if (strcmp(name,"natoms") == 0) return (void *) &lmp->atom->natoms;
+  if (strcmp(name,"nbonds") == 0) return (void *) &lmp->neighbor->nbondlist;
```

Comment:
this change is incorrect. the value of "nbonds" is stored in atom->nbonds
nbondlist, are the number of entries in the **local** list of bonds, which is different when running in parallel.